### DISCUSSION TITLE
Update beatmap online status colours inline with web

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#14681] [Ideas] [answered]
It is necessary to synchronize the colors of the status of maps from osu-web to the game
--
![ScreenShot_20210908143201](https://user-images.githubusercontent.com/86934170/132502435-c4865401-b1da-4aac-860b-e10debc01e4d.png)
![ScreenShot_20210908143206](https://user-images.githubusercontent.com/86934170/132502439-01c5e527-9437-4d1a-bd5b-2e2cef531d2d.png)
![ScreenShot_20210908143210](https://user-images.githubusercontent.com/86934170/132502441-ae6291cb-8b7b-4613-a358-c09a1f8c8272.png)
![ScreenShot_20210908143216](https://user-images.githubusercontent.com/86934170/132502443-bb71aec3-33fc-4a50-baf0-c6c947e47ffa.png)
![ScreenShot_20210908143222](https://user-images.githubusercontent.com/86934170/132502444-2be3e9df-e666-4532-8f25-69281c36d93b.png)
![ScreenShot_20210908143436](https://user-images.githubusercontent.com/86934170/132502452-519cda14-c680-44e2-b7cd-4ecc99925282.png)

**Comment by smoogipoo** [comment, marked as answer]
The entire song select UI will be redesigned (https://www.figma.com/file/DXKwqZhD5yyb1igc3mKo1P/Song-Select-Draft-2?node-id=0%3A1).

**Reply by Loreos7** [reply]
WOW.
